generators/test/index.js

```diff
@@ -0,0 +1,101 @@
+const Generator = require("yeoman-generator");
+const _ = require("lodash");
+const chalk = require("chalk");
+const fg = require("fast-glob");
```

Comment:
We have glob installed as a dependency - where is fast-glob?  Should we migrate to use glob for now and update them all to use fast-glob?

Reply:
Ah, my fault. I used that over in WebRH so I used it here. I've since switched it to regular `glob`. I don't think what we're doing would see a distinguishable difference with fast-glob.